### PR TITLE
add hsop-8 2.3x2.3mm ep

### PR DESCRIPTION
**Pull request by matthuszagh**
Adds a 2.3x2.3mm EP HSOP-8 version for [L7980](https://www.st.com/resource/en/datasheet/l7980.pdf).

### CODE REVIEW

File: scripts/Packages/Package_Gullwing__QFP_SOIC_SO/size_definitions/hsop.yaml [hunk 1]
```diff
@@ -48,3 +48,46 @@ HSOP-8-1EP_3.9x4.9mm_P1.27mm_EP2.41x3.1mm_ThermalVias:
...
+    tolerance: 0.1
+  body_size_y:
+    nominal: 4.9
+    tolerance: 0.1
+  overall_size_x:
+    nominal: 6
+    tolerance: 0.2
+  lead_width:
+    nominal: 0.41
+    tolerance: 0.1
```
Comment (chschlue) [inline scripts/Packages/Package_Gullwing__QFP_SOIC_SO/size_definitions/hsop.yaml:65]: No explicit nominal dimension given, please use min/max notation.

File: scripts/Packages/Package_Gullwing__QFP_SOIC_SO/size_definitions/hsop.yaml [hunk 2]
```diff
@@ -48,3 +48,46 @@ HSOP-8-1EP_3.9x4.9mm_P1.27mm_EP2.41x3.1mm_ThermalVias:
...
+  lead_len:
+    nominal: 0.65
+    tolerance: 0.15
```
Comment (chschlue) [inline scripts/Packages/Package_Gullwing__QFP_SOIC_SO/size_definitions/hsop.yaml:68]: Wrong dimensions. Also, see above.

File: scripts/Packages/Package_Gullwing__QFP_SOIC_SO/size_definitions/hsop.yaml [hunk 3]
```diff
@@ -48,3 +48,46 @@ HSOP-8-1EP_3.9x4.9mm_P1.27mm_EP2.41x3.1mm_ThermalVias:
...
+  num_pins_x: 0
+  num_pins_y: 4
+
+
+  EP_size_x:
+    nominal: 2.3
+    tolerance: 0.1
+  EP_size_y:
+    nominal: 2.3
+    tolerance: 0.1
```
Comment (chschlue) [inline scripts/Packages/Package_Gullwing__QFP_SOIC_SO/size_definitions/hsop.yaml:79]: No EP tolerance given.

**Review by chschlue — CHANGES_REQUESTED** [review]

**Comment by matthuszagh** [comment]
Thanks for the review @chschlue! I've implemented your changes. Let me know if any issues.

**Review by chschlue — APPROVED** [review]
Thanks.

**Pull request merged**